frontend/src/lib/components/TestConnection.svelte

```diff
@@ -19,7 +19,14 @@
 				content: `
 import { Client } from 'https://deno.land/x/postgres/mod.ts'
 export async function main(args: any) {
-	const client = new Client("postgres://" + args.user + ":" + args.password + "@" + args.host + ":" + (args.port ?? 5432) + "/" + args.dbname + "?sslmode=" + args.sslmode)
+	const u = new URL(`postgres://${args.user}:${args.password}@${args.host}:${args.port ?? 5432}/${args.dbname}?sslmode=${args.sslmode}`)
```

Comment:
I wonder if there's anything that's stopping us with using [`ClientOptions`](https://deno.land/x/postgres@v0.17.0/mod.ts?s=ClientOptions) here?

The only reason that I came up with was some backwards compatibility with `args` interface. Simple adapter like this could work:
```diff
-import { Client } from 'https://deno.land/x/postgres/mod.ts'
+import { Client, ClientOptions } from 'https://deno.land/x/postgres/mod.ts';
 export async function main(args: any) {
-       const client = new Client("postgres://" + args.user + ":" + args.password + "@" + args.host + ":" + (args.port ?? 5432) + "/" + args.dbname + "?sslmode=" + args.sslmode)
+       const clientOptions = {
+               database: args.dbname,
+               port: args.port ?? 5432,
+               hostname: args.host,
+               password: args.password,
+               user: args.user,
+       };
+       const client = new Client(clientOptions)
        await client.connect()
        return 'Connection successful'
 }
```

however there would be issues with mapping [`TLSOptions`](https://deno.land/x/postgres@v0.17.0/mod.ts?s=TLSOptions) to [psql `sslmode`](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION)